Defs/ThingDefs_Misc/Tiberium_Apparel_Hats.xml

```diff
@@ -21,7 +21,7 @@
 
 	<ThingDef ParentName="HatMakeableBase">
 		<defName>TiberiumSuitMask_TBP</defName>
-		<label>Tiberium radiation suit mask</label>
+		<label>TBP suit mask</label>

```

Comment:
TBP Acronym is not intuitive. Simply calling it "Tiberium Suit Mask" works, as it implies its function easily.